content/applications/general/developer_mode.rst

```diff
@@ -4,76 +4,82 @@
 Developer Mode (debug mode)
 ===========================
 
-The developer mode (or debug mode) gives you access to extra and advanced tools.
+The developer mode (or debug mode) unlocks access to extra and advanced tools.
```

Comment:
- Add `in Odoo` after `tools`
- Add a sentence `` There are several ways to activate the developer mode: through the :guilabel:`Settings` module, the browser extension, or the URL. `` after this sentence `The developer...advanced tools.`